### PR TITLE
add products.add endpoint

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**
### Added
- `/products.add`

### CODE REVIEW

**Review by dcsg — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -2618,6 +2618,22 @@ Get a list of products.
...
+
+     + Attributes (object)
+        + name: `cookies` (string, required)
+        + external_id (string, optional)
+
+ + Response 201 (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + data (object)
+            + type: `product` (string)
```
Comment (tiagobastos) [inline apiary.apib:2635]: Q: Shouldn't an identifier be returned too? 
```suggestion
            + type: `product` (string)
            + id: `ee94b4c0-5786-0517-9d26-8dd1e6406f20` (string)
```

**Review by tiagobastos — DISMISSED** [review]

**Comment by tiagobastos** [comment]
Didn't mean to approve.

File: src/06-products/products.apib [hunk 2]
```diff
@@ -24,3 +24,19 @@ Get a list of products.
...
+
+     + Attributes (object)
+        + name: `cookies` (string, required)
+        + external_id (string, optional)
+
+ + Response 201 (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + data (object)
+            + type: `product` (string)
```
Comment (steftrenson) [inline src/06-products/products.apib:42]: B: an id needs to be returned here as well.

**Review by steftrenson — DISMISSED** [review]

**Review by jdrieghe — APPROVED** [review]

File: src/06-products/products.apib [hunk 3]
```diff
@@ -24,3 +24,20 @@ Get a list of products.
...
                 + external_id: `123` (string)
+
+### products.add [POST /products.add]
+
+ Add a new product.
+
+ + Request (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + name: `cookies` (string, required)
```
Comment (mrtus) [inline src/06-products/products.apib]: https://giphy.com/gifs/rachaelrayshow-funny-crazy-BWoUbBTz1yuhW05Sgp

**Review by mrtus — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by luisaterrabrito — APPROVED** [review]

**Comment by jdrieghe** [comment]
Going to merge this one. Will align with @WouterSioen in a follow-up if we can expose similar behaviour in the `/companies.add` endpoint.

**Pull request merged**